src/core/config/config.rs

```diff
@@ -555,7 +555,7 @@ pub struct Http {
 
     #[serde(rename = "batchKey", default, skip_serializing_if = "is_default")]
     /// The `batchKey` parameter groups multiple data requests into a single call. For more details please refer out [n + 1 guide](https://tailcall.run/docs/guides/n+1#solving-using-batching).
-    pub group_by: Vec<String>,
+    pub group_by: Vec<serde_json::Value>,
```

Comment:
```suggestion
    pub batch_key: Option<String>,
    pub batch_path: Vec<String>,
```

So the user can specify both. `batch_path` and `batch_key`.